Vagrantfile

```diff
@@ -0,0 +1,41 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure("2") do |config|
+  config.vm.box = "hashicorp/bionic64"
+  config.vm.network "forwarded_port", guest: 5000, host: 8080
+  # config.vm.network "forwarded_port", guest: 5000, host: 5000 
+  
+
+  config.vm.provider "virtualbox" do |vb|
+      # Display the VirtualBox GUI when booting the machine
+      vb.memory = "2048"
+      vb.cpus = 4
+    end
+  
+  config.vm.provision "shell", privileged: false, inline: <<-SHELL
+    sudo apt-get update;
+    sudo apt-get install -y make build-essential libssl-dev zlib1g-dev libbz2-dev libreadline-dev libsqlite3-dev wget curl llvm libncursesw5-dev xz-utils tk-dev libxml2-dev libxmlsec1-dev libffi-dev liblzma-dev git python-openssl
+    git clone https://github.com/pyenv/pyenv.git ~/.pyenv
+    echo 'export PYENV_ROOT="$HOME/.pyenv"' >> ~/.profile
+    echo 'export PATH="$PYENV_ROOT/bin:$PATH"' >> ~/.profile
+    echo 'eval "$(pyenv init --path)"' >> ~/.profile
+    source ~/.profile
+    CFLAGS=-I/usr/include/openssl \
+    LDFLAGS=-L/usr/lib64 \
```

Comment:
I know you had some fun trying to get this working, but in our final implementation we can get rid of these two lines